cmake/dts.cmake

```diff
@@ -17,7 +17,6 @@ set(GENERATED_DTS_BOARD_CONF      ${PROJECT_BINARY_DIR}/include/generated/genera
 set(DTS_POST_CPP                  ${PROJECT_BINARY_DIR}/${BOARD}.dts.pre.tmp)
 
 set_ifndef(DTS_SOURCE ${BOARD_DIR}/${BOARD}.dts)
-set_ifndef(DTS_COMMON_OVERLAYS ${ZEPHYR_BASE}/dts/common/common.dts)
```

Comment:
Please replace this with something that will let the user know what happened.

```
if(DEFINED DTS_COMMON_OVERLAYS)
  message(FATAL_ERROR "DTS_COMMON_OVERLAYS is no longer supported. Use DTS_OVERLAY_FILE instead."
endif()
```

Reply:
Fixed